content/docs/using-pulumi/crossguard/snyk-policy.md

```diff
@@ -0,0 +1,256 @@
+```
+
+{{< notes type="info" >}}
+Note that the Snyk container policy is a [stack policy](docs/using-pulumi/crossguard/core-concepts/#stack-policies). This means that in order to have containers correctly scanned during a `pulumi preview`, your `docker.Image` resources must have the `buildOnPreview` set to `true`. If you run the policy pack during a `pulumi update`, the policy will execute _after_ your containers have been built, _and after they have been pushed_ if you have not explicitly specified the `skipPush` input to be `true`.
+{{< /notes >}}
+
+## Configuration options
+
+### Configuration parameters
+
+The Synk policy template supports the following configurable options:
+
+- `dockerfileScanning` (boolean): If set to `true`, Snyk will also scan your image's Dockerfile for additional remediation suggestions. If this value is set, `pulumiProgramAbsPath` must also be set. Default value is `false`.
+- `excludeBaseImageVulns` (boolean): If set to `true`, Snyk will exclude vulnerabilities in the base image for your Dockerfile as defined in the `FROM` directive, and limit its results to layers added on top of the base image. Default value is `false`.
+- `failOn` (enum): If set to `upgradable`, the image will be in violation only for vulnerabilities that have a known upgrade path. If set to `all`, the image will be in violation for all vulnerabilities. Default value is `all`.
+- `pulumiProgramAbsPath` (string): The absolute path on disk to the Pulumi program to which this policy is being applied.Required (and only useful) if `dockerfileScanning` is set to `true`. Default value is an empty string.
+- `severityThreshold` (enum): The minimum level of vulnerabilities to be reported, and if any are found, to find the image in violation of the policy. Valid values are `"low"`, `"medium"`, `"high"`, `"critical"`. Default value is `critical`.
+
+### Setting configuration values
+
+To set the configuration, you can do one of the following:
+
+1. Hard-code the values in `index.ts` within the templated policy. (These values will serve as deafults and may still be overridden on a per-run basis by either of the methods that follow.)
+1. Create a JSON file with the configuration values, e.g.:
+
+    ```json
+    {
+        "snyk-container-scan": {
+            "enforcementLevel": "disabled",
+            "dockerfileScanning": true,
+            "pulumiProgramAbsPath": "/path/to/pulumi/iac/program"
+        },
+    }
+    ```
+
+    And then pass that value to the Pulumi CLI when running the policy, e.g.:
+
+    ```bash
+    pulumi preview --policy-pack /path/to/policy --policy-pack-config /path/to/policy-config.json
+    ```
+
+    (Relative paths in the above command will also work.)
+1. If using Pulumi Cloud's server-side enforcement, policy pack configuration can be set centrally managed in the Pulumi Cloud UI. For details, see [Configuring policy packs](https://www.pulumi.com/docs/using-pulumi/crossguard/configuration/#using-the-pulumi-cloud).
```

Comment:
Language here is a bit off because the first two bullets are verbs ("Hard-code", "Create"). Suggest a slight tweak:

```suggestion
1. If using Pulumi Cloud, set the configuration values in the Pulumi Cloud console. For details, see [Configuring policy packs](/docs/using-pulumi/crossguard/configuration/#using-the-pulumi-cloud).
```

Reply:
Gonna pass on this suggestion b/c you may be using Pulumi Cloud as a backend, but you must be specifically paying for Pulumi Cloud to use server-side enforcement.